### ISSUE TITLE
rancher2_app_v2 version not available after installation of catalog

### ISSUE THREAD

**Issue by mikekuzak** [rancher/terraform-provider-rancher2#564]
TF 0.13.5
Rancher 2.5.3
Provider 1.11.0

When building a cluster it looks like the new versions of the apps are not available. But I can use these versions when I have a running cluster and upgrade to that version. 

```
module.k8s_cluster.rancher2_catalog_v2.catalogs_v2["bitnami"]: Creation complete after 5s [id=c-snwd9.bitnami]
module.k8s_cluster.rancher2_app_v2.applications["rancher-cis-benchmark"]: Creating...
module.k8s_cluster.rancher2_app_v2.applications["rancher-logging"]: Creating...
module.k8s_cluster.rancher2_app_v2.applications["longhorn"]: Creating...
Error: failed to get chart info rancher-logging:3.8.201 from catalog v2 rancher-charts: Bad response statusCode [500]. Status [500 Internal Server Error]. Body: [code=ServerError, message=no chart version found for rancher-logging-3.8.201] from [https://k8s-uat/k8s/clusters/c-snwd9/v1/catalog.cattle.io.clusterrepos/rancher-charts?link=info&chartName=rancher-logging&version=3.8.201]
  on .terraform/modules/k8s_cluster/main.tf line 299, in resource "rancher2_app_v2" "applications":
 299: resource "rancher2_app_v2" "applications" {
Error: failed to get chart info rancher-cis-benchmark:1.0.300 from catalog v2 rancher-charts: Bad response statusCode [500]. Status [500 Internal Server Error]. Body: [code=ServerError, message=no chart version found for rancher-cis-benchmark-1.0.300] from [https://k8s-uat/k8s/clusters/c-snwd9/v1/catalog.cattle.io.clusterrepos/rancher-charts?link=info&chartName=rancher-cis-benchmark&version=1.0.300]
  on .terraform/modules/k8s_cluster/main.tf line 299, in resource "rancher2_app_v2" "applications":
 299: resource "rancher2_app_v2" "applications" {
Error: failed to get chart info longhorn:1.1.000 from catalog v2 rancher-charts: Bad response statusCode [500]. Status [500 Internal Server Error]. Body: [code=ServerError, message=no chart version found for longhorn-1.1.000] from [https://k8s-uat/k8s/clusters/c-snwd9/v1/catalog.cattle.io.clusterrepos/rancher-charts?link=info&chartName=longhorn&version=1.1.000]
  on .terraform/modules/k8s_cluster/main.tf line 299, in resource "rancher2_app_v2" "applications":
 299: resource "rancher2_app_v2" "applications" {
ERROR: Job failed: exit status 1

```

**Comment by mikekuzak** [comment]
Same with Cluster Monitoring .. 
Rancher V2.5.5
TF Provider 1.11.0

```
module.k8s_cluster.rancher2_cluster.cluster-rke: Creating...
Error: Bad response statusCode [422]. Status [422 Unprocessable Entity]. Body: [message=cluster [c-m58jx] status version is not available yet. Cannot validate kube version for template [system-library-rancher-monitoring-0.2.0], baseType=error, code=InvalidBodyContent] from [https://my-rancher/v3/clusters/c-m58jx?action=enableMonitoring]
  on .terraform/modules/k8s_cluster/main.tf line 70, in resource "rancher2_cluster" "cluster-rke":
  70: resource "rancher2_cluster" "cluster-rke" {
```

When using monitoring 0.1.4 is works. 

Thanks

**Comment by rawmind0** [comment]
> When building a cluster it looks like the new versions of the apps are not available. But I can use these versions when I have a running cluster and upgrade to that version.

This issue should be addressed with PR https://github.com/rancher/terraform-provider-rancher2/pull/569

> Same with Cluster Monitoring ..

This is not the same issue. This is expected behaviour as explained at https://github.com/rancher/terraform-provider-rancher2/commit/669121c63787beb3828fa157f7712cd13c85201f

**Comment by rawmind0** [comment]
Fix included on new provider release v1.12.0

Please, reopen issue if needed
